packages/sdk/src/StreamrClient.ts

```diff
@@ -748,6 +749,7 @@ export class StreamrClient {
 
     /**
      * @deprecated This in an internal method
+     * @hidden
      */
     getConfig(): StrictStreamrClientConfig {
```

Comment:
Is there a good reason we've made this internal? I think it could be useful in practice to see the config that is in use?

Reply:
We can make it public in a follow-up pr if needed. Earlier discussion: https://github.com/streamr-dev/network/pull/1859#discussion_r1323122647